src/main/java/org/osiam/resources/exception/RestExceptionHandler.java

```diff
@@ -32,18 +32,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.MethodArgumentNotValidException;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.ResponseStatus;
-import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
 @ControllerAdvice({
         "org.osiam.auth.oauth_client",
         "org.osiam.metrics.controller",
         "org.osiam.resources.controller"
 })
-public class RestExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
BTW: The conflicting method is `org.osiam.resources.exception.RestExceptionHandler#handleValidationException`, but I think it will be fine anyways. There's still #48 open, that deals with the exception handling.